QRCoder/Base64QRCode.cs

```diff
@@ -43,25 +33,81 @@ public string GetGraphic(int pixelsPerModule, string darkColorHtmlHex, string li
+
+                byte[] pngData;
+                if (darkColor == Color.Black && lightColor == Color.White)
+                {
+                    pngData = pngCoder.GetGraphic(pixelsPerModule, drawQuietZones);
+                }
+                else
+                {
+                    byte[] darkColorBytes;
+                    byte[] lightColorBytes;
+                    if (darkColor.A != 255 || lightColor.A != 255)
+                    {
+                        darkColorBytes = new byte[] { darkColor.R, darkColor.G, darkColor.B, darkColor.A };
+                        lightColorBytes = new byte[] { lightColor.R, lightColor.G, lightColor.B, lightColor.A };
+                    }
+                    else
+                    {
+                        darkColorBytes = new byte[] { darkColor.R, darkColor.G, darkColor.B };
+                        lightColorBytes = new byte[] { lightColor.R, lightColor.G, lightColor.B };
+                    }
+                    pngData = pngCoder.GetGraphic(pixelsPerModule, darkColorBytes, lightColorBytes, drawQuietZones);
+                }
+
+                return Convert.ToBase64String(pngData, Base64FormattingOptions.None);
             }
-            return base64;
+
+#if NETFRAMEWORK || NETSTANDARD2_0 || NET5_0 || NET6_0_WINDOWS
+            if (
+#if NET6_0_OR_GREATER
+                RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+#else
+                true
+#endif
+                )
+            {
+                var qr = new QRCode(QrCodeData);
+                var base64 = string.Empty;
+                using (Bitmap bmp = qr.GetGraphic(pixelsPerModule, darkColor, lightColor, drawQuietZones))
+                {
+                    base64 = BitmapToBase64(bmp, imgType);
+                }
+                return base64;
+            }
+            else
+            {
+                throw new PlatformNotSupportedException("The specified image type is not supported on this platform.");
+            }
+#else
+            throw new PlatformNotSupportedException("The specified image type is not supported on this platform.");
+#endif
         }
 
+#if NETFRAMEWORK || NETSTANDARD2_0 || NET5_0 || NET6_0_WINDOWS
+#if NET6_0_WINDOWS
+        [System.Runtime.Versioning.SupportedOSPlatform("windows")]
+#endif
         public string GetGraphic(int pixelsPerModule, Color darkColor, Color lightColor, Bitmap icon, int iconSizePercent = 15, int iconBorderWidth = 6, bool drawQuietZones = true, ImageType imgType = ImageType.Png)
         {
+            var qr = new QRCode(QrCodeData);
             var base64 = string.Empty;
             using (Bitmap bmp = qr.GetGraphic(pixelsPerModule, darkColor, lightColor, icon, iconSizePercent, iconBorderWidth, drawQuietZones))
             {
                 base64 = BitmapToBase64(bmp, imgType);
             }
             return base64;
         }
+#endif
 
-
+#if NETFRAMEWORK || NETSTANDARD2_0 || NET5_0 || NET6_0_WINDOWS
+#if NET6_0_WINDOWS
```

Comment:
oh yeah, this is wrong.

Reply:
honestly the method won't exist on the `net6.0` TFM so it doesn't actually make a difference.  but so that the entire library is done the same way, i'll change it.